README.md

```diff
@@ -7,393 +7,88 @@
-```bash
-$ kubectl exec -ti cassandra-xxxxx -- nodetool status
-Datacenter: DC1
-===============
-Status=Up/Down
-|/ State=Normal/Leaving/Joining/Moving
---  Address          Load       Tokens       Owns (effective)  Host ID                               Rack
-UN  172.xxx.xxx.xxx  109.28 KB  256          50.0%             6402e90d-7995-4ee1-bb9c-36097eb2c9ec  Rack1
-UN  172.xxx.xxx.xxx  196.04 KB  256          51.4%             62eb2a08-c621-4d9c-a7ee-ebcd3c859542  Rack1
-UN  172.xxx.xxx.xxx  114.44 KB  256          46.2%             41e7d359-be9b-4ff1-b62f-1d04aa03a40c  Rack1
-UN  172.xxx.xxx.xxx  79.83 KB   256          52.4%             fb1dd881-0eff-4883-88d0-91ee31ab5f57  Rack1
-```
-
-
-
-# 5. Using CQL
-> **Note:** It can take around 5 minutes for the Cassandra database to finish its setup.
+### 1. Create a Cassandra Headless Service
 
-You can check if the Cassandra nodes are up and running by using this command:
-**Substitute the Pod name to the one you have**
-```bash
-$ kubectl exec cassandra-xxxxx -- nodetool status
-Datacenter: DC1
-===============
-Status=Up/Down
-|/ State=Normal/Leaving/Joining/Moving
---  Address          Load       Tokens       Owns (effective)  Host ID                               Rack
-UN  172.xxx.xxx.xxx  109.28 KB  256          50.0%             6402e90d-7995-4ee1-bb9c-36097eb2c9ec  Rack1
-UN  172.xxx.xxx.xxx  196.04 KB  256          51.4%             62eb2a08-c621-4d9c-a7ee-ebcd3c859542  Rack1
-UN  172.xxx.xxx.xxx  114.44 KB  256          46.2%             41e7d359-be9b-4ff1-b62f-1d04aa03a40c  Rack1
-UN  172.xxx.xxx.xxx  79.83 KB   256          52.4%             fb1dd881-0eff-4883-88d0-91ee31ab5f57  Rack1
-```
-> You will need to wait for the status of the nodes to be Up and Normal (UN) to execute the commands in the next steps.
-
-
-You can access the cassandra container using the following command:
-
-```bash
-$ kubectl exec -it cassandra-xxxxx /bin/bash
-root@cassandra-xxxxx:/# ls
-bin  boot  dev	docker-entrypoint.sh  etc  home  initial-seed.cql  lib	lib64  media  mnt  opt	proc  root  run  sbin  srv  sys  tmp  usr  var
-```
+To allow us to do simple discovery of the cassandra seed node (which we will deploy shortly) we can create a "headless" service.  We do this by  specifying **none** for the  **clusterIP** in the [cassandra-service.yaml](cassandra-service.yaml). This headless service  allows us to use KubeDNS for the Pods to discover the IP address of the Cassandra seed.
 
-Now run the sample .cql file to create and update employee table on cassandra keyspace using the following commands:
-> You only need to run the .cql file **once** in **ONE** Cassandra node. The other pods should also have access to the sample table created by the .cql file.
+You can create the headless service using the [cassandra-service.yaml](cassandra-service.yaml) file:
 
 ```bash
```

Comment:
While we're pokey poking around in here, we should change all of the `bash`s to `shell`.